HARP-2023-Summer/Mapping/Dataframe_Generator.Rmd

```diff
@@ -192,15 +192,14 @@ if(featr_type == "facility"){ #get facility-level fiveyr_avg_mgy from foundatn_m
                    by.x="featureid", by.y="facility_hydroid", all=T
                    )
   #join facility geometry to merged data frame:
-  featrs <- sqldf(paste("SELECT", #selects/renames desired columns from f_merge, matches them with geometry from f_geo based on featureid
-                        " a.facility, a.featureid as Facility_hydroid, a.Use_Type, a.locality, 
-                      a.sum as five_yr_avg, a.wsp_2040_mgy, a.Permit_Limit_MGY, a.",df$runlabel[1],", a.",df$runlabel[2],",
-                      b.",fn_geoCol(f_geo), #robust for varying geometry column names
+  featrs <- sqldf(paste("SELECT a.*, b.",fn_geoCol(f_geo), #robust for varying geometry column names
```

Comment:
This include all the columns that were obtained in the prior call to `om_vahydro_metric_grid()`, which is good since we wouldn't be able to include any other tabular data in the facility model results tables.